src/App.js

```diff
@@ -1,12 +1,23 @@
 import React, {Component} from 'react';
 import NavBar from "./components/Nav";
```

Comment:
While not required by the language spec, your component names as imported should generally match the class name, which should match the file name (unless there's a name conflict).